### ISSUE TITLE
Realy interesting... Can this be made dynamic?

### ISSUE THREAD

**Issue by ceased-ebc** [ayoungprogrammer/readAI#4]
Hi, love the look of this, could it be made dynamic? i.e feed in text and dump results in db's ?

**Comment by ayoungprogrammer** [comment]
Yes it can be made dynamic, you just need to change the inputs/outputs

**Comment by ceased-ebc** [comment]
Ah apologies, just noticed this response.

Question. 

Can this be made so it can scour and work from databases ? = 1PB of data and be asked questions and respond appropriately.

**Comment by ayoungprogrammer** [comment]
Yes I've applied this kinda of approach to answer questions with data sourced from WikiData and DBpedia. Look at my other repo: [NLQuery](http://github.com/ayoungprogrammer/nlquery). There are more sophisticated information retrieval methods for reading text and storing the structured data in a database. You can read more about that [here](https://nlp.stanford.edu/IR-book/).

**Comment by ceased-ebc** [comment]
Many thanks, apologies, no notifications,

I'd be interested in connecting it up to over 100 million tables over 500 databases, do you see this as feasible, just to have a play with?

**Comment by chendi1991** [comment]
Do you support Chinese?
